SampleApp/src/PortAudioMicrophoneWrapper.cpp

```diff
@@ -128,9 +172,48 @@ bool PortAudioMicrophoneWrapper::initialize() {
     }
     return true;
 }
+#ifdef XMOS_AVS_TESTS
+void PortAudioMicrophoneWrapper::ReaderThread(PortAudioMicrophoneWrapper *wrapper)
+{
+    ACSDK_LOG(alexaClientSDK::avsCommon::utils::logger::Level::INFO, alexaClientSDK::avsCommon::utils::logger::LogEntry("FileInput", "threadAlive"));
+    signed short block[128];
+    const unsigned period = sizeof(block) / 2 * 1000 / (unsigned)SAMPLE_RATE;
+    while (wrapper->m_threadFuture->wait_for(std::chrono::milliseconds(period)) == std::future_status::timeout) {
+        wrapper->m_fileStream->read((char*)block, sizeof(block));
+        if (wrapper->m_fileStream->fail() || wrapper->m_fileStream->eof()) {
+            if (!wrapper->m_eofReached) {
+                ACSDK_LOG(alexaClientSDK::avsCommon::utils::logger::Level::INFO, alexaClientSDK::avsCommon::utils::logger::LogEntry("FileInput", "eofReached"));
```

Comment:
A read failure not at EOF receives a log entry of "eofReached".  Is that the intention?

Reply:
yes, to stop processing of an audio track we send a quit command from the console.